### ISSUE TITLE
Require cache bust in watch mode

### ISSUE THREAD

**Issue by zellwk** [11ty/eleventy#325]
My `.eleventy.js` file looks like this: 

```js
const form = require('./eleventy/shortcodes/form')

module.exports = eleventyConfig => {
  eleventyConfig.addShortcode('formInput', form.input)
  eleventyConfig.addShortcode('formTextarea', form.textarea)
  eleventyConfig.addShortcode('formRadios', form.radios)
})
```

I wanted to write shortcodes in a separate file because they're too many to put into the eleventy.js config file. The problem with this approach is eleventy's `--watch` mode doesn't perform cachebust (unless I'm mistaken?). 

So, any updates I make to `./eleventy/shortcodes/form` doesn't get watched. I'll have to restart the cli manually.

**Comment by zachleat** [comment]
Ah, good call! This has implications for local dependencies used in #118 templates too (which I’m working on now).

Seems like https://www.npmjs.com/package/dependency-tree might be a good match for this if the speed is right?

**Comment by zachleat** [comment]
(For the record there are two things being reported here—we do require cache-bust on the config file `.eleventy.js` but do not require cache-bust or watch dependencies used in the config file yet)

**Comment by zachleat** [comment]
No worries @zellwk just thinking out loud! 👍

**Comment by zachleat** [comment]
Inspired by and in addition to this, I opened a feature request at #333. Upvote if you like that!

**Comment by zachleat** [comment]
This will be included with upcoming v0.7.0.

**Comment by bpeab** [comment]
I might be wrong but it still doesn't work even with `watchTarget` addition.
